### PR TITLE
Update tr.m3u

### PR DESCRIPTION
**Pull request by Emilselimli**
1.The non-working link of channels have been removed and replaced (Streamlock iplink closed)
2. 'artiyerelmedya' replaced 'www.artidijitalmedya.com' all channel links replaced
3. Dersim 62 TV closed channels

Add new channels
Number 1 ask
Number 1 damar
Number 1 dance

### CODE REVIEW

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
tvg-logo doesn't go here. If you need to change the logos, you need to change the [database entries](https://github.com/iptv-org/database) that are linked to this channel.

The rest looks okay to me at first view.

**Comment by Emilselimli** [comment]
[BellezaEmporium] bugs fixed

File: streams/tr.m3u [hunk 1]
```diff
@@ -469,3 +481,4 @@ https://cdn-kanaltvo.yayin.com.tr/kanaltvo/kanaltvo/playlist.m3u8
 https://s01.vpis.io/ogun/ogun.m3u8
 #EXTINF:-1 tvg-id="",Torba TV (720p) [Not 24/7]
 https://59cba4d34b678.streamlock.net/canlitv/torbatv/playlist.m3u8
+
```
Comment (BellezaEmporium) [inline streams/tr.m3u:484]:
```suggestion
```

**Review by BellezaEmporium — DISMISSED** [review]
Perfect, LGTM.

File: streams/tr.m3u [hunk 2]
```diff
@@ -448,17 +406,71 @@ https://uskudarunv.mediatriple.net/uskudarunv/uskudar1/playlist.m3u8
...
-#EXTINF:-1 tvg-id="VuslatTV.tr",Vuslat TV (720p) [Not 24/7]
-https://waw2.artiyerelmedya.net/vuslattv/bant1/playlist.m3u8
+#EXTINF:-1 tvg-id="MalatyaVuslatTV.tr",Vuslat TV (720p) 
+https://live.artidijitalmedya.com/artidijital_vuslattv/vuslattv/playlist.m3u8
+#EXTINF:-1 tvg-id="Kanal3.tr",KANAL 3
+https://live.artidijitalmedya.com/artidijital_kanal3/kanal3/playlist.m3u8
+#EXTINF:-1 tvg-id="",KONYA OLAY
+https://live.artidijitalmedya.com/artidijital_konyaolaytv/konyaolaytv/playlist.m3u8
+#EXTINF:-1 tvg-id="GaziantepOlayTV.tr",GAZİANTEP OLAY
+https://live.artidijitalmedya.com/artidijital_olaytv/olaytv/chunks.m3u8
```
Comment (UltraHDR) [inline streams/tr.m3u:419]: Please use playlist.m3u8 instead

Comment (freearhey) [inline streams/tr.m3u:419]:
```suggestion
https://live.artidijitalmedya.com/artidijital_olaytv/olaytv/playlist.m3u8
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/tr.m3u [hunk 3]
```diff
@@ -267,11 +223,15 @@ http://46.4.193.238:8484/hls/ntvspor/playlist.m3u8
...
-http://waw1.artiyerelmedya.net:1935/olayturk/bant1/playlist.m3u8
-#EXTINF:-1 tvg-id="OlayTurkTV.tr",Olay Türk TV (720p) [Geo-blocked]
-https://waw2.artiyerelmedya.net/olayturk/bant1/playlist.m3u8
+https://b01c02nl.mediatriple.net/videoonlylive/mtkgeuihrlfwlive/u_stream_5c9e17cd6360b_1/playlist.m3u8
+#EXTINF:-1 tvg-id="Number1Ask.tr",Number 1 ASK
+https://b01c02nl.mediatriple.net/videoonlylive/mtkgeuihrlfwlive/u_stream_5c9e18f9cea15_1/playlist.m3u8
+#EXTINF:-1 tvg-id="Number1Damar.tr",Number 1 DAMAR
+https://b01c02nl.mediatriple.net/videoonlylive/mtkgeuihrlfwlive/u_stream_5c9e198784bdc_1/playlist.m3u8
+#EXTINF:-1 tvg-id="Number1Dance.tr",Number 1 DANCE
+https://b01c02nl.mediatriple.net/videoonlylive/mtkgeuihrlfwlive/u_stream_5c9e2aa8acf44_1/playlist.m3u8
```
Comment (freearhey) [inline streams/tr.m3u:232]: "Number 1" channels are already in your other request #12906

**Comment by freearhey** [comment]
Closed due to lack of activity.

**Pull request closed without merging**